### PR TITLE
RockLibrary endpoints code fix up

### PR DESCRIPTION
**Pull request by Mars-98**
had to fix if else statements, as well as create a dictionary for debugging purposes
https://tim-kaufenberg.atlassian.net/browse/ARC-99

### CODE REVIEW

**Comment by coughenberg** [comment]
Fix merge conflicts, it looks like it happened because you weren't completely up to date with the rock-library branch when you submitted the pr. Let me know if you need help fixing this

**Comment by Mars-98** [comment]
got it fixed, lmk if there's anything else needed to be done

File: rocklibrary/rocklibrary_endpoint/recieve_image_endpoint.py [hunk 1]
```diff
@@ -38,24 +38,31 @@ def getImageUploadAndColor():
     
     
     colorRequest = request.forms.get('color')
+    print(colorRequest)
```
Comment (coughenberg) [inline rocklibrary/rocklibrary_endpoint/recieve_image_endpoint.py]: Delete print statement

File: rocklibrary/rocklibrary_endpoint/recieve_image_endpoint.py [hunk 2]
```diff
@@ -38,24 +38,31 @@ def getImageUploadAndColor():
...
-    else:
+    if userHeightRequest is None:
         userHeightRequest = ''
-
-    wallHeightRequest = request.forms.get('wallHeight')
-    if wallHeightRequest is not None and not wallHeightRequest.isnumeric():
-        abort(400, 'Bad wall height number input')
     else:
+        try:
+            user_HEIGHT = float(userHeightRequest)
```
Comment (coughenberg) [inline rocklibrary/rocklibrary_endpoint/recieve_image_endpoint.py]: I assumed you checked this, but I figured I have to check, if you input a int does this break converting to float?

File: rocklibrary/rocklibrary_endpoint/recieve_image_endpoint.py [hunk 3]
```diff
@@ -70,6 +77,14 @@ def getImageUploadAndColor():
     im = Image.open(file_path)
     im.show()
 
+    endPts = {
+        "color":colorRequest,
+        "image":imageRequest,
+        "userHeight":userHeightRequest,
+        "wallHeight":wallHeightRequest
+    }
+    print(endPts)
```
Comment (coughenberg) [inline rocklibrary/rocklibrary_endpoint/recieve_image_endpoint.py]: Delete print statement

File: rocklibrary/rocklibrary_endpoint/recieve_image_endpoint.py [hunk 4]
```diff
@@ -70,6 +77,14 @@ def getImageUploadAndColor():
     im = Image.open(file_path)
     im.show()
 
+    endPts = {
+        "color":colorRequest,
+        "image":imageRequest,
+        "userHeight":userHeightRequest,
+        "wallHeight":wallHeightRequest
+    }
```
Comment (coughenberg) [inline rocklibrary/rocklibrary_endpoint/recieve_image_endpoint.py]: Same here as I commented in the other endpoint pr. If not appearing then do not send back and space after :, along with the comment put here
https://github.com/ARC-Summer-2020/AutoRockClimber/pull/16/files#r480688710

File: rocklibrary/rocklibrary_endpoint/recieve_image_endpoint.py (hunk 2 above)
Comment (coughenberg) [reply rocklibrary/rocklibrary_endpoint/recieve_image_endpoint.py]: Also don't put the variable with an underscore following height in all caps, follow the other naming conventions on this page

File: rocklibrary/rocklibrary_endpoint/recieve_image_endpoint.py [hunk 5]
```diff
@@ -38,24 +38,31 @@ def getImageUploadAndColor():
...
+            user_HEIGHT = float(userHeightRequest)
+        except:
+            abort(400, 'Bad user height number input')
+        
+    wallHeightRequest = request.forms.get('wallHeight')
+    if wallHeightRequest is None:
         wallHeightRequest = ''
+    else:
+        try:
+            wall_HEIGHT = float(wallHeightRequest)
```
Comment (coughenberg) [inline rocklibrary/rocklibrary_endpoint/recieve_image_endpoint.py]: Same here with naming convention

**Review by coughenberg — CHANGES_REQUESTED** [review]

File: rocklibrary/rocklibrary_endpoint/recieve_image_endpoint.py (hunk 2 above)
Comment (Mars-98) [reply rocklibrary/rocklibrary_endpoint/recieve_image_endpoint.py]: Yes, I did check with casting to an int, it works fine. and okay I completely forgot about naming conventions lol, my b

File: rocklibrary/rocklibrary_endpoint/recieve_image_endpoint.py [hunk 6]
```diff
@@ -44,18 +44,24 @@ def getImageUploadAndColor():
...
+            abort(400, 'Bad user height number input')
+        
+    wallHeightRequest = request.forms.get('wallHeight')
+    if wallHeightRequest is None:
         wallHeightRequest = ''
+    else:
+        try:
+            wHeight = float(wallHeightRequest)
+        except:
+            abort(400, 'Bad wall height number input')
```
Comment (skitscha) [inline rocklibrary/rocklibrary_endpoint/recieve_image_endpoint.py]: Can we move this to its own  method so its more reusable?

File: rocklibrary/rocklibrary_endpoint/recieve_image_endpoint.py [hunk 7]
```diff
@@ -70,8 +76,19 @@ def getImageUploadAndColor():
     im = Image.open(file_path)
     im.show()
 
+    endPts = {}
```
Comment (skitscha) [inline rocklibrary/rocklibrary_endpoint/recieve_image_endpoint.py]: Can we rename this to something a little more intuitive like params?

**Review by coughenberg — APPROVED** [review]

File: rocklibrary/rocklibrary_endpoint/recieve_image_endpoint.py (hunk 6 above)
Comment (Mars-98) [reply rocklibrary/rocklibrary_endpoint/recieve_image_endpoint.py]: fixed it

File: rocklibrary/rocklibrary_endpoint/recieve_image_endpoint.py (hunk 7 above)
Comment (Mars-98) [reply rocklibrary/rocklibrary_endpoint/recieve_image_endpoint.py]: done

**Review by skitscha — APPROVED** [review]

**Pull request merged**